Remotion/Data/DomainObjects.UnitTests/Persistence/Rdbms/SqlServer/IntegrationTests/SqlProviderGeneratedSqlTest.cs

```diff
@@ -81,13 +83,17 @@ public void LoadDataContainers_SingleID ()
     [Test]
```

Comment:
please add test with empty collection.

Reply:
Unfortunately, this does not execute any SQL command, because in `LookupCommandFactory.CreateForSortedMultiIDLookup()`, the `objectIDList` is empty, resulting ina `MultiObjectLoadCommand<DataContainer?>` with an empty collection of `dbCommandBuildersAndReaders`, wherefore nothing is done in `MultiObjectLoadCommand<T>.Execute()`.